### PR TITLE
Fork out a suggested global-default from catch-all to handle unclassified flows

### PR DESCRIPTION
**Pull request by yue9944882**
/sig api-machinery
/kind feature

```release-note
NONE
```

### CODE REVIEW

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm

**Comment by MikeSpreitzer** [comment]
@MikeSpreitzer

**Comment by MikeSpreitzer** [comment]
@jktomer

File: staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go [hunk 1]
```diff
@@ -161,7 +160,7 @@ var (
 					Queuing: &flowcontrol.QueuingConfiguration{
 						Queues:           64,
 						HandSize:         6,
-						QueueLengthLimit: 1000,
+						QueueLengthLimit: 10,
```
Comment (lavalamp) [inline staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go]: 10 seems a bit on the small side, if we consider that many of our requests, especially read ones, are served in < 1ms. I think 50 or 100 is an OK starting point? (Apologies if we talked about this at the meeting and I said something different, I can't remember!)

File: staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go [hunk 2]
```diff
@@ -80,22 +83,17 @@ var (
...
+					Type: flowcontrol.LimitResponseTypeReject,
 				},
 			},
 		})
 )
 
 // Mandatory FlowSchema objects
 var (
-	// exempt priority-level
+	// "exempt" priority-level is used for preventing priority inversion.
```
Comment (lavalamp) [inline staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go]: And for ensuring that sysadmin requests are always possible.

File: staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go (hunk 1 above)
Comment (yue9944882) [reply staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go]: yeah we talked about that in the meeting and agreed the current queue lengths are exaggeratedly long but we didnt decide another actual number. applied 50 instead.

File: staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go (hunk 2 above)
Comment (yue9944882) [reply staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go]: updated comments

**Comment by lavalamp** [comment]
Thanks for changes! A test is angry, but here's an approve so you can get an LGTM in your timezone when you fix it:

/approve

**Comment by MikeSpreitzer** [comment]
I would be happy to work on fixing the failing test, but am missing a github clue.  Is there a way that I can add a commit to this PR?

**Comment by MikeSpreitzer** [comment]
The problem is that a change to validation_test.go has been merged while this PR was open, and the latest validation_test.go is inconsistent with the change made here.  See https://github.com/MikeSpreitzer/kubernetes/tree/refactor/catch-all for a rebased version of this change that updates validation_test.go too.

**Comment by yue9944882** [comment]
close in favor of #87528

**Pull request closed without merging**